src/irImage_msg.cpp

```diff
@@ -37,15 +37,15 @@ IRImageMsg::IRImageMsg() {}
 IRImageMsg::IRImageMsg(const std::shared_ptr<aditof::Camera> &camera,
                        aditof::Frame **frame, std::string encoding) {
     imgEncoding = encoding;
-    FrameDataToMsg(camera, frame);
+   // FrameDataToMsg(camera, frame);
```

Comment:
I don't see why these line should be commented. They are meant to update the message

Reply:
On second thought I think it is correct this way as well. Just don't leave it commented, you can remove the lines entirely

Reply:
remove those lines, it is addressed in the most recent commit.